docker-compose.yml

```diff
@@ -1,12 +1,24 @@
 services:
-  shell:
+  prod-shell:
+    image: front-end-monorepo_prod:latest
+    build:
+      context: ./
+      args:
+        - NODE_ENV=production
+        - PANOPTES_ENV=production
+        - NEXT_TELEMETRY_DISABLED=1
+        - APP_ENV=development
+    command:
+      - "/bin/sh"
+  dev-shell:
     image: front-end-monorepo_dev:latest
-    volumes:
-      - node_modules:/usr/src/node_modules
```

Comment:
I removed this to make it easier to debug the node package installs. Otherwise, you're just looking at the `node_modules` directory from your local machine.